unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs

```diff
@@ -365,28 +375,28 @@ void UpdateParcelHighlight()
 
         void UpdateParcelHold()
         {
-            if (Vector2.Distance(lastClickedCursorMapCoords, cursorMapCoords) > MAX_CURSOR_PARCEL_DISTANCE / (scaleFactor * 2.5f))
-            {
-                OnCursorFarFromParcel?.Invoke();
-            }
+            if (Vector2.Distance(lastClickedCursorMapCoords, cursorMapCoords) > MAX_CURSOR_PARCEL_DISTANCE / (scaleFactor * 2.5f)) { OnCursorFarFromParcel?.Invoke(); }
         }
 
-        private void OnKernelConfigChanged(KernelConfigModel current, KernelConfigModel previous) { validWorldRanges = current.validWorldRanges; }
+        private void OnKernelConfigChanged(KernelConfigModel current, KernelConfigModel previous)
+        {
+            validWorldRanges = current.validWorldRanges;
+        }
 
         bool CoordinatesAreInsideTheWorld(int xCoord, int yCoord)
         {
             foreach (WorldRange worldRange in validWorldRanges)
             {
-                if (worldRange.Contains(xCoord, yCoord))
-                {
-                    return true;
-                }
+                if (worldRange.Contains(xCoord, yCoord)) { return true; }
             }
+
             return false;
         }
 
         private void MapRenderer_OnSceneInfoUpdated(MinimapMetadata.MinimapSceneInfo sceneInfo)
         {
+            Debug.Log($"VV: {Time.frameCount} CALL = {scenesOfInterest.Contains(sceneInfo)}");
```

Comment:
Maybe wrap this under some VERBOSE flag so it doesn't hit production?
```suggestion
            #ifdef VERBOSE
            Debug.Log($"VV: {Time.frameCount} CALL = {scenesOfInterest.Contains(sceneInfo)}");
            #endif
```

Reply:
Good call, anyway normal logs are disabled in production anyway, but let's keep the editor console clean as well.